cu-webteam/d8-platform
# Feature/ckeditor media module change button images

# Description

New custom module created and related configuration to turn it on by default. The new module works to change CKEDITOR buttons icon image through changing the data structure in the modules js/change.js file.

## Related PRs

N/A

## Todos

N/A

## Deploy Notes

N/A

## Steps to Test or Reproduce

Test A:

1. Go to the configuration for CKEDITOR. example: Configuration -> Content authorizing -> Text formats and editors.
2. Select Configure button on a CKEDITOR text editor
3. Look for the media browser button and look to see if the icon is changed to a icon with a mountain silhouette.

<img width="1039" alt="screen shot 2019-03-05 at 12 14 35 am" src="https://user-images.githubusercontent.com/5378479/53784908-57e71500-3edc-11e9-8098-ac11f6148c4f.png">

Test B:

1. Add a content page: example: Content -> Add content -> Content page
2. Check to see if the media browser icon image is changed to an icon with a mountain silhouette.

<img width="675" alt="screen shot 2019-03-05 at 12 16 20 am" src="https://user-images.githubusercontent.com/5378479/53784961-7e0cb500-3edc-11e9-9150-65f684c7fc92.png">
